docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_xl.mdx

```diff
@@ -156,31 +164,52 @@ refiner = DiffusionPipeline.from_pretrained(
 refiner.to("cuda")
 ```
 
-Now we define the number of inference steps and the fraction at which the model shall be run through the 
+Now we define the number of inference steps and the point at which the model shall be run through the 
 high-noise denoising stage (*i.e.* the base model).
 
 ```py
 n_steps = 40
-high_noise_frac = 0.7
+high_noise_transition = 200  # or high_noise_frac = 0.8
 ```
 
-A fraction of 0.7 means that 70% of the 40 inference steps (28 steps) are run through the base model
-and the remaining 12 steps are run through the refiner. Let's run the two pipelines now.
-Make sure to set `denoising_end` and `denoising_start` to the same values and keep `num_inference_steps`
-constant. Also remember that the output of the base model should be in latent space:
+Stable Diffusion XL base is trained on timesteps 0-999 and Stable Diffusion refiner is finetuned
+from the base on timesteps 0-200, so we use the base model for the first 800 timesteps (high noise) 
+and the refiner for the last 200 timesteps (low noise). Hence, `high_noise_transition`
+is set to 200, so that all steps 200-999 are performed by the base model and steps 0-199 are
+performed by the refiner model.
```

Comment:
We mention that the first 800 steps will be performed by the base model but here we're specifying 200-999, which are not the first 800 steps? 

What am I missing out on?

Reply:
Inference works backwards through the noising process, so timesteps 999 -> 0, iteratively. 999 = highest noise, 0 = lowest noise.

Reply:
My bad. Sorry.